### PR TITLE
Some structure updates

### PR DESCRIPTION
**Pull request by SteveHNH**

### CODE REVIEW

**Review by aleccohan — APPROVED** [review]
This is awesome!  Just one little thing.  With these changes I "think" we're going to have to change where the Procfile is pointing

i.e. `web: gunicorn server:app`  --> `web: gunicorn app:app`

**Review by wcmitchell — APPROVED** [review]
LGTM!

File: app/handlers/routes.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    @app.route("/", methods=["GET"])
+    def root():
+        return "lubdub"
+
+    @app.route("/booze", methods=["GET"])
+    def booze():
+        result = firebase.fb.get("/", None)
+        return Response(response=json.dumps(result), mimetype="application/json")
+
+    @app.route("/drinks", methods=["GET"])
```
Comment (aleccohan) [inline app/handlers/routes.py:26]: @sadams This is looking awesome!  One little thing, it looks like something is up the `/drinks` endpoint.  I'm getting a wild error:
`127.0.0.1 - - [17/Aug/2021 11:44:52] code 400, message Bad request version ('À\x13À')
127.0.0.1 - - [17/Aug/2021 11:44:52] "÷óÌjk±Û
                                             ½9!AûzÞ¿óñ£)¢÷- -¶ì[@k+¬@IoDÅ¿uBÈÊFáÀê³/À+À0À,Ì©Ì¨À        ÀÀ" HTTPStatus.BAD_REQUEST -`
                                             
Currently looking for what might be causing this issue!  All other endpoints are working great though!

Comment (aleccohan) [reply app/handlers/routes.py:26]: @sadams actually ignore me! I "think" this issue is being caused by us trying to hit an `https://` endpoint from a local `http://` server!

**Pull request merged**